_posts/2011-12-15-nsq_internals.md

```diff
@@ -218,14 +218,14 @@ is the wide range of debugging capabilities it supports. The [`net/http/pprof`][
 package integrates directly with the native HTTP server, exposing endpoints to retrieve CPU, heap,
 goroutine, and OS thread profiles. These can be targeted directly from the `go` tool:
 
-    $ go tool pprof http://127.0.0.1:4151/debug/pprof/profile
+    $ go tool pprof https://127.0.0.1:4151/debug/pprof/profile
```

Comment:
not here ;)

Reply:
@ploxiln finding errors in my PRs since 2011. Throwback eh? cc @mreiferson @jehiah

Reply:
(... and similarly for the many other 127.0.0.1 links)

Reply:
I always have a fully validated self signed cert from CA_sricola to serve local https traffic. but fine fine, fixed.